next/src/app/layout.tsx

```diff
@@ -1,11 +1,14 @@
 import { Kanit } from 'next/font/google';
+import '../assets/global.scss';
 
 const font = Kanit({ weight: '400', subsets: ['latin'] });
```

Comment:
Change `font` const name for corresponding fontFamily name.
In this case `Kanit`